modules/metastorage/src/test/java/org/apache/ignite/internal/metastorage/server/AbstractCompactionKeyValueStorageTest.java

```diff
@@ -510,6 +510,18 @@ void testGetSingleEntryAndCompactionForFooKey() {
         assertDoesNotThrowCompactedExceptionForGetSingleValue(FOO_KEY, 5);
     }
 
+    @Test
+    void testDoNotCompactOnExactMatch() {
```

Comment:
So we actually do not want to compact on exact match? Should we update the javadoc?

Reply:
I'll change the name of the test, I guess it turned out to be misleading, I understand it now